### PR TITLE
Add locketd example and fix mistakes

### PR DESCRIPTION
**Pull request by alanrme**

### CODE REVIEW

File: docs/administration/configuration.md [hunk 1]
```diff
@@ -28,7 +29,19 @@ security:
...
     key_file: "" # The optional location of an SSL key to use
 locket:
-    token: extremely secure password # The password to use to secure the `/lockets` endpoint
+    token: extremely secure password # The password used to secure the `/lockets` endpoint
 prometheus:
-    token: extremely secure password # The password to use to secure the `/metrics` endpoint
+    token: extremely secure password # The password used to secure the `/metrics` endpoint
+```
+
+### locketd.yml
```
Comment (alecks) [inline docs/administration/configuration.md:37]: Headers should be followed by an empty line.

**Review by alecks — CHANGES_REQUESTED** [review]

**Comment by XilogOfficial** [comment]
# FIXED.

# pls merge

File: docs/administration/configuration.md [hunk 2]
```diff
@@ -28,7 +29,19 @@ security:
...
+```yaml
+environment: release # Enables verbose logging (debug or release)
+port: 8000 # Port for locketd
+security:
+    rh4NaXhju914cn60CHmuMREeQG1Qdh53o4sQ9iZWVlA= # A secure 32 byte key; try `openssl rand -base64 32`
```
Comment (alecks) [inline docs/administration/configuration.md:44]: Maybe specify that it has to be the same as Cell's.

**Review by alecks — CHANGES_REQUESTED** [review]

File: docs/administration/configuration.md (hunk 2 above)
Comment (XilogOfficial) [reply docs/administration/configuration.md:44]: bruh wait

Comment (alecks) [reply docs/administration/configuration.md:44]: `security` also isn't a string; you want `security.secret`.

**Review by alecks — CHANGES_REQUESTED** [review]

**Comment by XilogOfficial** [comment]
# FIXED

# MERGE.

**Review by alecks — APPROVED** [review]

**Comment by XilogOfficial** [comment]
THANK U

**Pull request merged**